### PR TITLE
Product listing

### PR DESCRIPTION
**Pull request by sanu29**
image

### CODE REVIEW

File: src/component/Filter.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        
+           
+
+           {categories.map((item)=>{
+              return (
+                <li className="margin-8" key={item.id}>
+                <label>  
+                    <input type="checkbox" name="category" className="input input-checkbox" 
+                    id={item.categoryName}/>
+                    {(item.categoryName)}
```
Comment (aiman-ansari) [inline src/component/Filter.jsx:38]: here you can destructure categoryName from item

File: src/component/ProductBody.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import { Categories, Img } from "./images"
+import { useCategoriesContext } from "../context/categories-context"
+import {ThreeDots} from "react-loader-spinner";
+import {Link} from "react-router-dom";
+import { useProductContext } from "../context/products-context";
+
+
+export const Products = () =>{
+    const {products} = useProductContext()
+    console.log(products)
```
Comment (aiman-ansari) [inline src/component/ProductBody.jsx:10]: Remove console log

File: src/component/AllComponents.jsx [hunk 3]
```diff
@@ -1,3 +1,6 @@
 export {Footer} from "./Footer"
 export {Header} from "./Header"
 export {HomepageBody} from "./HomepageBody"
+export {Products} from "./ProductBody"
+export {Filter} from "./Filter"
+
```
Comment (aiman-ansari) [inline src/component/AllComponents.jsx:6]: Instead of AllComponent you can name it as index.js

File: src/component/Filter.jsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import { useCategoriesContext } from "../context/categories-context"
+import {ThreeDots} from "react-loader-spinner";
+export const Filter = () =>{
+    const {categories} = useCategoriesContext();
+    console.log(categories)
```
Comment (zoef-yt) [inline src/component/Filter.jsx:5]: remove console.log()

**Review by aiman-ansari — APPROVED** [review]
code is nicely written and easy to understand also the thing you need to do is work more on your code indentation .
check the mentioned suggestions.

**Review by zoef-yt — APPROVED** [review]
The code looks good. The UI looks great. you are good to merge.

**Review by tonymike99 — APPROVED** [review]
Hi Saniya,
Most of the code looks good!
Lot of files though because of the pictures xD
Anyway, good to go

**Pull request merged**